src/pages/ImportData.js

```diff
@@ -107,6 +108,7 @@ const ImportData = () => {
   )(async workspace => {
     const namespace = workspace.namespace
     const name = workspace.name
+    let success = true
```

Comment:
The logic using this boolean is spread pretty widely. I wonder if we can tighten it up.

Reply:
Pushed up a solution for it, just going to throw an error and let that catch it